integration/apps/sinatra2_classic/docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: '3.4'
+services:
+  app:
+    build:
+      context: .
+      args:
+        BASE_IMAGE: datadog/dd-apm-demo:rb-2.7
+    depends_on:
+      - ddagent
+    environment:
+      - BUNDLE_GEMFILE=/app/Gemfile
+      - DD_AGENT_HOST=ddagent
+      - DD_METRIC_AGENT_PORT=8125
+      - DD_TRACE_AGENT_PORT=8126
+      - DD_HEALTH_METRICS_ENABLED=true
+      - DD_SERVICE=acme-sinatra
+      - DD_PROFILING_ENABLED=true
+      # Use these to choose what is run
+      - DD_DEMO_ENV_PROCESS=puma
+      - DD_DEMO_ENV_FEATURES=tracing,profiling
+      # Use this for a local version of ddtrace
+      - DD_DEMO_ENV_GEM_LOCAL_DDTRACE=/vendor/dd-trace-rb
+      # Use these for a specific revision of ddtrace
+      # - DD_DEMO_ENV_GEM_GIT_DDTRACE=https://github.com/DataDog/dd-trace-rb.git
+      # - DD_DEMO_ENV_GEM_REF_DDTRACE=f233336994315bfa04dac581387a8152bab8b85a
+      # Enable building the profiling native extension
+      # - DD_DEMO_ENV_BUILD_PROFILING_EXTENSION=true
```

Comment:
Ideally, I'd like this enabled, to validate the profiler works as well. Did you run into any issues with this enabled?